doc/classes/String.xml

```diff
@@ -669,9 +669,10 @@
 			<argument index="2" name="maxsplit" type="int" default="0">
 			</argument>
 			<description>
-				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				Splits the string by a [code]divisor/delimiter[/code] string and returns an array of the substrings, starting from right.
+				The splits in the returned array are sorted in the same order as the original string, from left to right.
+				[b]Example:[/b] [code]"One,Two,Three,Four"[/code] will return [code]["Three","Four"][/code] if split by [code]","[/code] with maxsplit of [code]"2"[/code].
```

Comment:
I still think the example should come *after* the paragraph that defines what `maxsplit` is. Otherwise it's confusing that you mention maxsplit at all before defining it.

`maxsplit` should be formatted as code, as its example value is an integer, so it should not be quoted like a string (so either just `2` or `[code]2[/code]` if we want it to stand out).